Jetcaster/core/designsystem/src/main/java/com/example/jetcaster/designsystem/component/PodcastImage.kt

```diff
@@ -80,14 +78,7 @@ fun PodcastImage(
                         .fillMaxSize()
                 )
             }
-            else -> {
-                Box(
-                    modifier = Modifier
-                        .background(placeholderBrush)
-                        .fillMaxSize()
```

Comment:
I think this is currently part of the design :) So i think it needs to be left in. 
![Screenshot 2024-10-21 at 09 47 34](https://github.com/user-attachments/assets/ecce64ec-f40e-4233-abf2-18a33e49f6d4)

Reply:
As you can see, the background created in advance made the UX appear awkward during the image animation, so I removed it. Do you think a rollback is needed?

https://github.com/user-attachments/assets/c2481342-7841-4d63-aaeb-5e66116c09b3

 https://github.com/user-attachments/assets/8e2820b3-a300-403c-8d53-2fde4dd79b24